vamdctap-webservice/src/main/java/org/vamdc/tapservice/RequestProcess.java

```diff
@@ -123,6 +124,16 @@ public void setLastModified(Date date) {
 			lastModified = date;
 		}
 	}
+	
+	@Override
+	public Date getLastModified() {
+		return this.lastModified;
+	}
+	
+	@Override
+	public String getUUID(String requestType) {
+		return "basecol:"+UUID.randomUUID()+":"+requestType;
```

Comment:
Also, no basecol-specific strings here, please :)
If you want to take the node name, take it from Settings.